src/journal/compress.c

```diff
@@ -41,6 +41,61 @@ DEFINE_TRIVIAL_CLEANUP_FUNC(LZ4F_decompressionContext_t, LZ4F_freeDecompressionC
 #if HAVE_ZSTD
 DEFINE_TRIVIAL_CLEANUP_FUNC(ZSTD_CCtx *, ZSTD_freeCCtx);
 DEFINE_TRIVIAL_CLEANUP_FUNC(ZSTD_DCtx *, ZSTD_freeDCtx);
+
+static int zstd_ret_to_errno(size_t ret) {
+        switch (ZSTD_getErrorCode(ret)) {
+#if ZSTD_VERSION_NUMBER  >= 10301
+        case 70: /* ZSTD_error_dstSize_tooSmall */
```

Comment:
would really prefer if we'd use the symbolic names if we have them. Specifying literal numbers here is just too ugly for the common case...